### PR TITLE
Admin - Test - Update Menu

### PR DESCRIPTION
**Pull request by hungbkdn**

### CODE REVIEW

File: tests/Browser/AdminUpdateMenuTest.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+     *
+     * @return void
+     */
+    public function testUpdateSuccess()
+    {
+        $this->browse(function (Browser $browser) {
+            factory(Category::class, 1)->create()->each(function($c) {
+                $c->foods()->save(factory(Food::class)->make());
+            });
+            factory(DailyMenu::class, 1)->create(['food_id' => 1, 'date' => '2017-09-11']);
```
Comment (ghost) [inline tests/Browser/AdminUpdateMenuTest.php]: You create data with date = 2017-09-11. If run test after this date, your test will fail. 
You can try to use: `date('Y-m-d', strtotime(' +1 day'))`

**Review by ghost — COMMENTED** [review]
Please check my comment.
You should add cases for disable/enable update button. And case for cancel update button.

**Comment by hungbkdn** [comment]
I have updated, please check it again!
Thank you so much @at-vietpham

**Comment by ghost** [comment]
LGTM 🙇

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**